net/net.sh

```diff
@@ -128,13 +130,15 @@ while getopts "h?T:t:o:f:rD:i:c:" opt; do
   c)
     getClientTypeAndNum() {
       if ! [[ $OPTARG == *'='* ]]; then
-        echo "Error: Expecting keypair \"clientType=numClientType\" but got \"$OPTARG\""
+        echo "Error: Expecting tuple \"clientType=numClientType=extraArgs\" but got \"$OPTARG\""
```

Comment:
Update `usage` to reflect the ability to pass extra args.

Also, this parameter is getting complicated.  Might want a section in the readme to give folks examples of using it.

Reply:
Will update usage in the next patch. Sorry, didn't see the comment, and there was automerge label.

Reply:
@pgarg

Reply:
@jackcmay , you just tagged a different user (outside solana)

Reply:
Sweet, more eyes the better ;-)